daffodil-lib/src/main/scala/org/apache/daffodil/xml/QNameBase.scala

```diff
@@ -477,6 +489,15 @@ protected trait RefQNameFactoryBase[T] {
 
   protected def constructor(prefix: Option[String], local: String, namespace: NS): T
 
+  /**
+   * This variant to resolve normal QNames such as in a ref="pre:local" syntax, or
+   * a path step like these steps .../foo/bar:baz/quux.
+   *
+   * @param qnameString
+   * @param scope
+   * @param unqualifiedPathStepPolicy
+   * @return
+   */
```

Comment:
Empty scaladoc on the variables.

Reply:
Yeah, I hate these scaladoc redundant tag things. They're either empty, or not maintained properly because the code changes, and they do not. 

I am using Intellij IDEA now, and haven't go tthese things properly customized yet. There may be whitespace and indent things that are wrong too.